app/src/main/java/tech/relaycorp/letro/awala/AwalaManager.kt

```diff
@@ -132,23 +129,20 @@ class AwalaManagerImpl @Inject constructor(
     ) {
         withContext(awalaThreadContext) {
             if (outgoingMessage.type != MessageType.AuthorizeReceivingFromServer && awalaSetupJob != null) {
-                Log.i(TAG, "Awala wasn't initialized while tried to send a message. Wait for completion... $outgoingMessage")
+                logger.i(TAG, "Awala wasn't initialized while tried to send a message. Wait for completion... $outgoingMessage")
                 awalaSetupJob?.join()
-                Log.i(TAG, "Awala was initialized, proceed futher...")
+                logger.i(TAG, "Awala was initialized, proceed futher...")
             }
             val firstPartyEndpoint = loadFirstPartyEndpoint()
             val thirdPartyEndpoint = loadThirdPartyEndpoint(
                 sender = firstPartyEndpoint,
                 recipient = recipient,
             )
-            Log.i(TAG, "sendMessage() from ${firstPartyEndpoint.nodeId} to ${thirdPartyEndpoint.nodeId}: $outgoingMessage)")
-            GatewayClient.sendMessage(
-                OutgoingMessage.build(
-                    type = outgoingMessage.type.value,
-                    content = outgoingMessage.content,
-                    senderEndpoint = firstPartyEndpoint,
-                    recipientEndpoint = thirdPartyEndpoint,
-                ),
+            logger.i(TAG, "sendMessage() from ${firstPartyEndpoint.nodeId} to ${thirdPartyEndpoint.nodeId}: $outgoingMessage)")
```

Comment:
I know it's tangential to this PR, but we should stop logging the full `$outgoingMessage` for privacy/security reasons. How about we just log the content type?

Reply:
Sure!